app/src/main/res/layout/activity_pokemon_resources.xml

```diff
@@ -0,0 +1,237 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context="me.quadphase.qpdex.PokemonResources"
+    android:id="@+id/pokemon_resources"
+    android:orientation="vertical"
+    android:background="@drawable/pkmnrestile">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textAppearance="?android:attr/textAppearanceLarge"
+        android:text="For more information, please visit the following websites:"
+        android:id="@+id/textView"
+        android:autoText="false"
+        android:gravity="left"
+        android:layout_weight="0"
+        android:textColor="#ff5c6cff"
+        android:fontFamily=""
+        android:layout_marginBottom="10dp"
+        android:background="@color/dex_grey" />
+
+    <LinearLayout
```

Comment:
This LinearLayout is redundant since it encloses a ScrollView which contains a LinearLayout itself.